quisby/benchmarks/pyperf/pyperf.py

```diff
@@ -0,0 +1,70 @@
+
+from scipy.stats import gmean
+
+def custom_key(item):
+    if item[0] == "localhost":
+        return (item[0])
+    else:
+        instance_type = item[0].split("-")[0]
+        instance_number = int(item[0].split('-')[-1])
+        return (instance_type, instance_number)
+
+
+def create_summary_pyperf_data(data,OS_RELEASE):
+
+    results = [['SYSTEM',"N0_OF_TEST_PROCESSES","CPU_INTEGER_MATH", "CPU_FLOATINGPOINT_MATH", "CPU_PRIME", "CPU_SORTING", "CPU_ENCRYPTION", "CPU_COMPRESSION", "CPU_SINGLETHREAD", "CPU_PHYSICS", "CPU_MATRIX_MULT_SSE", "CPU_mm", "CPU_sse", "CPU_fma", "CPU_avx", "CPU_avx512", "m_CPU_enc_SHA", "m_CPU_enc_AES", "m_CPU_enc_ECDSA", "ME_ALLOC_S", "ME_READ_S", "ME_READ_L", "ME_WRITE", "ME_LARGE", "ME_LATENCY", "ME_THREADED", "SUMM_CPU", "SUMM_ME"]]
```

Comment:
If I'm reading this right it says "N0_OF_TEST_PROCESSES" rather than "NO_OF_TEST_PROCESSES" - note the zero instead of capital O.  Is that right?